islandora_marcxml.module

```diff
@@ -97,3 +105,17 @@ function islandora_marcxml_islandora_ingest_steps_alter(array &$steps, array &$f
     }
   }
 }
+
+/**
+ * Implements hook_islandora_edit_datastream_registry().
+ */
+function islandora_marcxml_islandora_edit_datastream_registry(AbstractObject $object, AbstractDatastream $datastream) {
+  if ($datastream->id == 'MODS') {
+    return array(
+      'islandora_marcxml_update_mods_from_marcxml' => array(
+        'name' => t('Update MODS from MARCXML'),
```

Comment:
Not everybody knows that MARCXML is not stored in an Object. Maybe `name` should be explicit about "from a user provided MARXML"  ?